hoodie-client/src/main/java/com/uber/hoodie/index/bloom/HoodieBloomIndex.java

```diff
@@ -303,7 +303,7 @@ public boolean isImplicitWithStorage() {
    * if we dont have key ranges, then also we need to compare against the file. no other choice if
    * we do, then only compare the file if the record key falls in range.
    */
-  private boolean shouldCompareWithFile(BloomIndexFileInfo indexInfo, String recordKey) {
+  boolean shouldCompareWithFile(BloomIndexFileInfo indexInfo, String recordKey) {
```

Comment:
Let's not change this ?

Reply:
@n3nash I see your point. Thought of that too. 
HoodieGlobalBloomIndex::explodeRecordRDDWithFileComparisons will be using this function as well. 
Shall I just copy this function / logic to HoodieGlobalBloomIndex?

Reply:
Ahh, I see, this is fine, thanks for the note.